docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -120,6 +130,61 @@ traces:
     enabled: false
 ```
 
+### Grafana Enterprise Traces helm chart values
+
+A complete example for GET is as follows.
+
+```yaml
+---
+global:
+  clusterDomain: 'cluster.local'
+
+multitenancyEnabled: true
+enterprise:
+  enabled: true
+  image:
+    tag: weekly-r83-4e9cb510
+enterpriseGateway:
+  enabled: true
+gateway:
+  enabled: false
+minio:
+  enabled: true
+storage:
+  trace:
+    backend: s3
+    s3:
+      access_key: 'grafana-tempo'
+      secret_key: 'supersecret'
+      bucket: 'enterprise-traces'
+      endpoint: 'tempo-minio:9000'
+      insecure: true
+  admin:
+    backend: s3
+    s3:
+      access_key_id: 'grafana-tempo'
+      secret_access_key: 'supersecret'
+      bucket_name: 'enterprise-traces-admin'
+      endpoint: 'tempo-minio:9000'
+      insecure: true
+traces:
+  otlp:
+    http:
+      enabled: true
+    grpc:
+      enabled: true
+distributor:
+  config:
+    log_received_spans:
+      enabled: true
+
+license:
+  contents: |
+    LICENSEGOESHERE
+```
+
+As noted above, please observe the additional `admin` bucket, the `gateway` has been disabled, the `enterpriseGateway` has been enabled, and a license has been specified.
+
 Next, we will:
 
 1. Create a `custom.yaml` file
```

Comment:
Please delete lines 188-192. I've moved this section earlier in the document.

Reply:
Okay, we had three places with that wording.  Double check me now, but it looks okay to me.